crates/matrix-sdk-base/src/client.rs

```diff
@@ -1325,6 +1355,13 @@ impl BaseClient {
             })
             .collect()
     }
+
+    /// Returns a receiver that gets events for each room info update. To watch
+    /// for new events, use `receiver.resubscribe()`. Each event contains the
+    /// room and a boolean whether this event should trigger a room list update.
+    pub fn roominfo_update_receiver(&self) -> &broadcast::Receiver<RoomInfoUpdate> {
+        &self.roominfo_update_receiver
```

Comment:
…because `self.roominfo_update_sender.subscribe()` will return an owned `Receiver`.

Then we can update the comment to not indicate to use of `resubscribe()`.